src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,350 @@
+  top: 0;
+  bottom: 0;
+  padding-top: var(--pf-c-sequence__control--PaddingTop);
+  padding-bottom: var(--pf-c-sequence__control--PaddingBottom);
+
+  &::before {
+    content: "";
+    border-top: var(--pf-c-sequence__control--BorderTopWidth) solid var(--pf-c-sequence__control--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--BorderBottomWidth) solid var(--pf-c-sequence__control--BorderBottomColor);
+  }
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopWidth) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomWidth) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopWidth) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomWidth) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopWidth) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomWidth) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control-m-start {
+  left: var(--pf-c-sequence__control-m-start--PositionLeft);
+  order: 1;
+  border-right: var(--pf-c-sequence__control-m-start--BorderRightWidth) solid var(--pf-c-sequence__control-m-start--BorderRightColor);
+  border-left: 0;
+}
+
+.pf-c-sequence__control-m-end {
+  right: var(--pf-c-sequence__control-m-end--PositionRight);
+  order: 3;
+  border-right: 0;
+  border-left: var(--pf-c-sequence__control-m-end--BorderLeftWidth) solid var(--pf-c-sequence__control-m-end--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  min-width: var(--pf-c-sequence__control-icon--MinWidth);
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  order: 2;
+  height: var(--pf-c-sequence__list--Height);
+  padding-bottom: var(--pf-c-sequence__list--PaddingBottom);
+  overflow-x: scroll;
+  counter-reset: sequence-number;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+
+  &:not(:last-child) {
+    margin-right: var(--pf-c-sequence__item--MarginRight);
+  }
+}
+
+// Content
+.pf-c-sequence__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: var(--pf-c-sequence__content--MinHeight);
+}
+
+// Content Title
+.pf-c-sequence__content-title {
+  max-width: var(--pf-c-sequence__content-title--MaxWidth);
```

Comment:
I've noticed that if I add a longer string of text to the title the number is no longer keeping it's size. I think this can be fixed by adjusting the max-width. Let me know if you want to chat about a fix for this.

Reply:
Increased the max-width on the title/status and the item to allow for larger strings of text. For now this follows the design, however if anyone feels strongly about having the width of the items be the same, then we can definitely revise.